integration_tests/tests/forseti/controls/explain.rb

```diff
@@ -23,79 +23,76 @@
 
 control "explain" do
   @inventory_id = /\"id\"\: \"([0-9]*)\"/.match(command("forseti inventory create --import_as #{random_string}").stdout)[1]
-  describe command("forseti model use #{random_string}") do
-    its('exit_status') { should eq 0 }
-  end
 
   # access_by_member
-  describe command("forseti explainer access_by_member serviceaccount/#{forseti_server_service_account}") do
+  describe command("forseti model use #{random_string} && forseti explainer access_by_member serviceaccount/#{forseti_server_service_account}") do
```

Comment:
Sometimes these SSH commands don't get run in the correct order, so the individual explain tests have to be coupled with the model use command to ensure the correct model is set.